### PR TITLE
chore(docs): add bundle size analyzer

### PR DESCRIPTION
**Pull request by nnixaa**
### Please read and mark the following check list before creating a pull request:

 - [] I read and followed the [CONTRIBUTING.md](https://github.com/akveo/nebular/blob/master/CONTRIBUTING.md) guide.
 - [] I read and followed the [New Feature Checklist](https://github.com/akveo/nebular/blob/master/DEV_DOCS.md#new-feature-checklist) guide.
 
 #### Short description of what this resolves:

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -174,7 +176,8 @@
     "typedoc": "^0.12.0",
     "typescript": "3.4.5",
     "uglify-es": "3.3.9",
-    "uglifyjs-webpack-plugin": "2.1.2"
+    "uglifyjs-webpack-plugin": "2.1.2",
```
Comment (yggg) [inline package.json]: Why also add `uglifyjs-webpack-plugin`?

File: package.json [hunk 2]
```diff
@@ -32,7 +32,9 @@
     "docs:prod": "npm run build -- docs --prod --aot --base-href '/nebular/'",
     "docs:dirs": "gulp create-docs-dirs",
     "docs:gh-pages": "npm run docs:build && npm run ngh -- --dir ./docs/dist",
-    "build": "ng build",
+    "docs:build:stats": "npm run docs:prod -- --stats-json",
+    "docs:analyze": "webpack-bundle-analyzer docs/dist/stats.json",
+    "build": "node --max_old_space_size=8048 ./node_modules/@angular/cli/bin/ng build",
```
Comment (yggg) [inline package.json]: Why do we need to call angular-cli this way?

**Comment by jondavidjohn** [comment]
I built a tool to help you analyze webpack bundles for size regressions, and report them directly to GitHub PRs. It's free for open source, so it might be worth checking out and helpful in this scenario.

[https://packtracker.io](https://packtracker.io/?utm_campaign=links&utm_source=github&utm_medium=issueprcomment)

image

File: package.json (hunk 1 above)
Comment (nnixaa) [reply package.json]: we didn't, the , was added, that's why there is a change here

File: package.json (hunk 2 above)
Comment (nnixaa) [reply package.json]: as `node` was running out of memory.

File: package.json (hunk 1 above)
Comment (yggg) [reply package.json]: sorry 🙂

**Pull request closed without merging**